src/test/java/serenityswag/inventory/AddProductsActions.java

```diff
@@ -0,0 +1,18 @@
+package serenityswag.inventory;
+
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.serenitybdd.core.steps.UIInteractionSteps;
+import net.thucydides.core.annotations.Step;
+
+import java.util.List;
+
+public class AddProductsActions extends UIInteractionSteps {
+
+    ProductList productList;
+    @Step("Add Products'")
+    public Integer forAllAvailableItems() {
+        List<WebElementFacade> itemsAdded = productList.availableItems();
```

Comment:
```suggestion
        List<WebElementFacade> addToCartButtons = productList.addToCartButtons();
```
name could be better i think. You are not returning the items you can buy but the button to click